skpy/chat.py

```diff
@@ -340,7 +340,7 @@ class SkypeGroupChat(SkypeChat):
             still appears in the recent list despite being left or deleted.
     """
 
-    attrs = SkypeChat.attrs + ("topic", "creatorId", "userIds", "adminIds", "open", "history", "picture", "active")
+    attrs = SkypeChat.attrs + ("topic", "creatorId", "userIds", "adminIds", "open", "history", "picture", "active", "moderatedthread")
```

Comment:
This should be `moderated` for consistency with the method to change it.  It also needs adding to the docstring.